packages/vue-client/tsconfig.app.json

```diff
@@ -4,9 +4,14 @@
   "exclude": ["src/**/__tests__/*"],
   "compilerOptions": {
     "composite": true,
-    "baseUrl": ".",
+    "baseUrl": "..",
+    "rootDir": "..",
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["vue-client/src/*"],
+      "@ogfcommunity/variants-shared": ["./shared/src/index"]
     }
+    ,
+    "verbatimModuleSyntax": false,
```

Comment:
I think `verbatimModuleSyntax` off makes it so I don't need to put `type { SomeImport }` **everywhere** in `shared/`.  I'm not opposed to it, but it would be quite a large change, and I wanted to keep this diff to config-only as much as possible.